### PR TITLE
Added Brazilian Portuguese on extension

### PR DESCRIPTION
**Pull request by amandafilizola**
Implements following issues: #53

## Changelog
- Brazilian Portuguese localization

### CODE REVIEW

**Review by XFox111 — CHANGES_REQUESTED** [review]
Thank you for the contribution!
I've reviewed the PR. There're some minor changes required to be made but overall quality is perfect (at least as far as I trust Google Translate). Great job!

File: manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "__MSG_name__",
-	"version": "2.0",
+	"version": "2.0.1",
```
[the remark this answers is not in the document]
Comment (amandafilizola) [reply manifest.json]: Updated.

File: _locales/pt_BR/messages.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	{
+		"message": "Reservar abas ao apertar o ícone da extensão (%TOGGLE_SHORTCUT% ou clique-direito para abrir o painel)",
+		"description": "Label for option"
+	},
+	"github":
+	{
+		"message": "Visite a página do GitHub",
+		"description": "Link title"
+	},
+	"contributors":
```
[the remark this answers is not in the document]
Comment (amandafilizola) [reply _locales/pt_BR/messages.json]: Updated.

**Review by XFox111 — APPROVED** [review]

**Comment by XFox111** [comment]
Thank you for being interested in my project! It's really important to me.
Your changes have been merged into 2.0 version branch and will be published as soon as the version will be ready to deliver

**Pull request merged**